frontend/src/routes.tsx

```diff
@@ -15,6 +15,9 @@ import {isAuthorized} from "./utils/common";
 import {useDispatch, useSelector} from "react-redux";
 import {RootState} from "./types/data";
 import {resetLogout} from "./actions/common";
+import Layout from "./components/Layout";
+import {Paragraph} from "./components/Paragraph";
```

Comment:
I don't think Paragraph is being used

Reply:
Same as Layout

Reply:
Done. Thanks for the reminder. I try to do it when I remember, but we don't have an automatic way to do this. Also many of my files show in red because of the `import React, {useEffect, useState} from "react";` import at the top. React is never used and Typescript doesn't like that.